### ISSUE TITLE
Block page refresh cmd+r / ctrl+r (maybe F5)

### ISSUE THREAD

**Issue by jdegger** [philhudson91/SwagChat#17]

**Comment by phil-hudson** [comment]
I think page refresh is important for people with bad internet connection or when simply things go bad, I find I use it on Franz quite a bit

**Comment by jdegger** [comment]
Agreed but it should not refresh the whole ui, just the selected webview imo. I will assign myself with the UI stuff I'm planning to do.
